src/components/Container/Container.styles.tsx

```diff
@@ -1,11 +1,14 @@
+import type { CSSProperties } from 'react';
 import styled, { css } from 'styled-components';
 import {
   mobileOnly,
   minHeightFix,
   minHeightFixChild,
 } from '../../core/GlobalStyle/mixin';
 
-export const Container = styled.div`
+export const Container = styled.div<{
+  direction: CSSProperties['flexDirection'];
```

Comment:
아래에서 'column'을 기본값으로 주는 걸 보니 optinal 해야 할 것 같습니다!

```suggestion
  direction?: CSSProperties['flexDirection'];
```

Reply:
@kimjeongwonn 스타일 쪽에서 기본 값을 처리하지 않고 기본값을 컴포넌트 쪽으로 올려서 처리하는 편이 보기 좋을거 같아서 끌어올리도록 하겠습니다!